packages/webapp/pages/devcard.tsx

```diff
@@ -282,7 +282,7 @@ const Step2 = ({
               defaultTypo={false}
               target="_blank"
             >
-              <GitHubIcon className="mr-2" />
+              <GitHubIcon className="mr-2 w-auto h-auto" />
```

Comment:
Did we not add a auto class?

Reply:
No, we don't and really I don't want to introduce this one - I think it's better to go with concrete sizes set through size prop, but now we have two places where GitHub icon is 16px and 17px (I already mentioned it in a Slack thread), so it's two new sizes for our icon component. So I prefer to go with className here and update it later when we will have a guideline for icon sizes.

Reply:
Current size here is 17px and to keep it 17px we need to set w-auto and h-auto, because height is set by wrapping element. And in account details GitHub icon is 16px. These are two new sizes and I'm not even sure that 17px is something that should exist, but I think it will be better to keep it as it is while we don't have a guideline